plugins/CMakeLists.txt

```diff
@@ -125,6 +126,11 @@ if(ENABLE_PLUGIN_M2K)
 	endif()
 endif()
 
```

Comment:
Firfilter resource files should be added in the installers.
Check the DACPlugin (for csv) or RegmapPlugin (for xmls) in order to implement a similar resource bundling mechanism for these filters. 
I think we should add these in the installer in their separate plugins. So, while having generic xmls or csvs for generic plugins (regmap, dac) - we should use specific directories for these plugins (examples: ad936x or ad9084 or whatever the plugin name is)